areas-of-d-usage.dd

```diff
@@ -36,8 +36,6 @@ $(DIVC start-icons,
 )
 
 $(DIV,
-    <img id="dman" src="images/d3.png" border="0" align="left" alt="D Man">
```

Comment:
"Professionalism" had no play in my decision :) There was just too much going on in that part of the page, as I already mentioned we use the same D-Man image on a few other pages already, and it did make the icon layout quite easier.